src/modules/events/events.js

```diff
@@ -29,3 +30,11 @@ bus.on('change', (change) => {
 		else winston.info('Events registered');
 	});
 });
+
+bus.on('http/init', app => {
+	app.use(route.post('/x/change', function *() {
```

Comment:
I don't think we should allow arbitrary change events from a rest endpoint. Also this shouldn't be in events module.